RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -1003,8 +1183,16 @@ void vtkSlicerRoomsEyeViewModuleLogic::UpdateGantryToFixedReferenceTransform(vtk
     vtkErrorMacro("UpdateGantryToFixedReferenceTransform: Invalid parameter set node");
     return;
   }
+  vtkSlicerBeamsModuleLogic* beamsLogic = vtkSlicerBeamsModuleLogic::SafeDownCast(this->GetModuleLogic("Beams"));
+  if (!beamsLogic)
+  {
+    vtkErrorMacro("UpdateGantryToFixedReferenceTransform: Beams logic cannot be accessed");
+    return;
+  }
 
   this->IECLogic->UpdateGantryToFixedReferenceTransform(parameterNode->GetGantryRotationAngle());
+  vtkMRMLLinearTransformNode* gantryToFixedReferenceTransformNode = beamsLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::Gantry, vtkSlicerIECTransformLogic::FixedReference);
+  gantryToFixedReferenceTransformNode->Modified();

```

Comment:
Explain in a comment (inline, at the end of the line) why you need to call `Modified`. Please do it for every `Modified` call in this commit (if there are several one after the other it's enough to do this for the first one but needed in different places and functions)